tests/wsgi_test.py

```diff
@@ -748,27 +748,29 @@ def wsgi_app(environ, start_response):
         result = read_http(sock)
         self.assertEqual(result.status, 'HTTP/1.1 417 Expectation Failed')
         self.assertEqual(result.body, b'failure')
-        fd.write(
-            b'PUT / HTTP/1.1\r\nHost: localhost\r\nContent-length: 7\r\n'
-            b'Expect: 100-continue\r\n\r\ntesting')
-        fd.flush()
-        header_lines = []
-        while True:
-            line = fd.readline()
-            if line == b'\r\n':
-                break
-            else:
-                header_lines.append(line)
-        assert header_lines[0].startswith(b'HTTP/1.1 100 Continue')
-        header_lines = []
-        while True:
-            line = fd.readline()
-            if line == b'\r\n':
-                break
-            else:
-                header_lines.append(line)
-        assert header_lines[0].startswith(b'HTTP/1.1 200 OK')
-        assert fd.read(7) == b'testing'
+
+        for expect_value in ('100-continue', '100-Continue'):
+            fd.write(
+                b'PUT / HTTP/1.1\r\nHost: localhost\r\nContent-length: 7\r\n'
+                b'Expect: %s\r\n\r\ntesting' % expect_value.encode())
```

Comment:
Please, never use `%` templating.
It's bad in general (because tuple special case) and in this particular case it broke py3.4 tests.

`'bla bla bla: {}'.format().encode()`